webpack.config.js

```diff
@@ -0,0 +1,39 @@
+var fs = require('fs');
+var path = require('path');
+var webpack = require('webpack');
+var banner = `/**
+ * @license
+ * Copyright (c) 2015 Google
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+`;
+
+module.exports = {
+  entry: {
+    'webvr-polyfill': './src/main.js',
+    'webvr-polyfill.min': './src/main.js',
+  },
+  output: {
+    path: path.join(__dirname, 'build'),
+    filename: '[name].js',
+  },
+  resolve: {
+    extensions: ['.js', '.js', '.json']
+  },
```

Comment:
if you add to this parent config object, `devtool: 'source-map',`, Webpack will build external source maps (tested them, and they work well)